### DISCUSSION TITLE
Crashed after pressing Details

### DISCUSSION THREAD

**Discussion by ariiiq** [TT-ReBORN/Georgia-ReBORN#72] [Q&A]
ive been trying to open up details to show the details of the songs, it crashed everytime. I did tried changing songs too and it didnt work, i tried changing the options for details in the Options menu and still crashed, not sure what happened but its been happening like crazy, the only fix i had right now is just not even clicking details at all, and if i did, had to restart the app all over again.

**Comment by TT-ReBORN** [comment]
Hi @ariiiq, 

please post the crash log here, can't do anything without knowing what's going on.
Btw, did you modify the files? You can also backup your files and try to reset the theme 
in top menu Options > Settings > Theme settings > Reset all.

-TT
